assets/helpers/contributions.js

```diff
@@ -286,6 +287,16 @@ function parseContrib(s: ?string, contrib: Contrib): Contrib {
   }
 }
 
+function parseRegularContributionType(s: ?string): RegularContributionType {
+  switch (s) {
```

Comment:
The switch-case feels odd, is there a better way of doing this? If we use an if we save two lines.

```
if ( s === 'ANNUAL') {
return s;
} else {
     return 'MONTHLY
}
```

Reply:
Good call.